compose/index.md

```diff
@@ -66,6 +67,28 @@ Compose has commands for managing the whole lifecycle of your application:
  * Stream the log output of running services
  * Run a one-off command on a service
 
+## Compose V2 and the new `docker compose` command
+
+> Important
+>
+> The new Compose V2, which supports the `compose` command as part of the Docker CLI, is now available.
+>
+> Compose V2 integrates compose functions into the Docker platform, continuing to support most of the previous `docker-compose` features and flags. You can test the Compose V2 by simply replacing the dash (`-`) with a space, and by running `docker compose`, instead of `docker-compose`.
+{: .important}
+
+Starting with Docker Desktop 3.4.0, you can run Compose V2 commands without modifying your invocations, by enabling the drop-in replacement of the previous `docker-compose` with the new command.  See the section [Installing Compose](install.md) for detailed instructions.
```

Comment:
```suggestion
If you rely on using Docker Compose as `docker-compose` (with a hyphen), you can set up Compose V2 to act as a drop-in replacement of the previous `docker-compose`.  Refer to the [Installing Compose](install.md) section for detailed instructions.
```

(for the above, wondering if we need an explicit "run compose v2 as drop-in replacement for v1" section, that outlines how to run it as `docker-compose` (with the alias / symlink)